tests/acceptance/config/behat.yml

```
@@ -174,6 +174,21 @@ default:
         - OcisConfigContext:
         - PublicWebDavContext:
 
+    apiFullTextSearch:
+      paths:
+        - '%paths.base%/../features/apiFullTextSearch'
```

create a new `apiSearch` suite and bring other search feature files into this suite as well.
```suggestion
    apiSearch:
      paths:
        - '%paths.base%/../features/apiSearch'
```

---

or we have already `fullSearch.feature` we can simply add this to that feature. No need to create new suite.

---

> create a new `apiSearch` suite and bring other search feature files into this suite as well.

@saw-jan IMHO, This suite is added for full-text search which needs Tika service. Other search scenarios don't need Tika service so it does not worth running Tika service for all search scenarios. So suggested to create a separate suite.

---

> or we have already `fullSearch.feature` we can simply add this to that feature. No need to create new suite.

@SagarGi `fullSearch.feature` contains scenarios related to the tag. Searching through tag won't need Tika service. The refactoring of the file name will be done in the next PR.

---

Ahh! That's right. At the time of reviewing this, there was no tika mentioned.
:+1: